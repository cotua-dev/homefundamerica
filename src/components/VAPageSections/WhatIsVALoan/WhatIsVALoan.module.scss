@import '../../../styles/variables';

section#whatIsVALoanSection {
    height: 60vh;
    display: flex;
    justify-content: center;
    background-color: $primaryColor;
    color: $textColor;

    @media screen and (max-width: 950px) {
        height: 100%;
        padding: 40px 10px;
    }

    div.contentWrapper {
        width: 600px;
        display: flex;
        align-items: flex-start;
        justify-content: center;
        flex-direction: column;

        @media screen and (max-width: 950px) {
            text-align: center;
        }

        @media screen and (max-width: 767px) {
            width: 100%;
        }

        h2.title {
            margin: 0;
            font-size: 32px;

            @media screen and (max-width: 950px) {
                font-size: 24px;
                text-align: center;
                width: 100%;
            }
        }

        p.content {
            line-height: 24px;
            margin: 10px 0;
            font-size: 16px;

            @media screen and (max-width: 950px) {
                font-size: 14px;
                text-align: center;
                width: 100%;
            }
        }
    }
}
