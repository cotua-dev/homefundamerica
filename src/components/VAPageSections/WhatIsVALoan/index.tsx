import React, {
    FunctionComponent,
    ReactElement,
} from 'react';
import * as styles from './WhatIsVALoan.module.scss';

export const WhatIsVALoan: FunctionComponent = (): ReactElement => {
    return (
        <section id={styles.whatIsVALoanSection}>
            <div className={styles.contentWrapper}>
                <h2 className={styles.title}>{`What Is a VA Loan?`}</h2>
                <p className={styles.content}>
                    {`
                        It's a home loan with no required down payment and
                        favorable loan terms for Active Duty Service members,
                        Veterans, Reservists, National Guard members, surviving
                        spouses, and certain other individuals who meet
                        eligibility requirements can use VA loans to purchase
                        homes or refinance existing mortgages. VA loans are
                        guaranteed by the Department of Veterans Affairs.
                    `}
                </p>
                <p className={styles.content}>
                    {`
                        VA loans are one of the only mortgages to offer 100%
                        financing (no down payment) for a home purchase. In
                        addition, VA loans offer refinance and cash out options
                        that can be used to pay off debt, fund major expenses,
                        make home improvements, or refinance a non-VA home loan
                        into a VA home loan with improved terms.
                    `}
                </p>
                <p className={styles.content}>
                    {`
                        There is also a streamlined option known as an Interest
                        Rate Reduction Refinance Loan (IRRRL) that provides
                        veterans with existing VA loans an opportunity to quickly
                        and easily refinance an existing VA loan. Underwriting
                        terms are very relaxed on an IRRRL an they allow veterans
                        or active duty to quickly and easily pursue a refinance
                        when market conditions improve and interest rates drop.
                    `}
                </p>
            </div>
        </section>
    );
}
