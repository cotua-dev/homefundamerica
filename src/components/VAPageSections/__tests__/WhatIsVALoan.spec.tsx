import React from 'react';
import {
    render,
    RenderResult,
} from '@testing-library/react';
import { WhatIsVALoan } from '../WhatIsVALoan';

describe('WhatIsVALoan Component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<WhatIsVALoan/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
