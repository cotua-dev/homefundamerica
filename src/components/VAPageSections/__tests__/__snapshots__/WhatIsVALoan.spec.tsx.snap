// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatIsVALoan Component renders 1`] = `
<section>
  <div>
    <h2>
      What Is a VA Loan?
    </h2>
    <p>
      
                        It's a home loan with no required down payment and
                        favorable loan terms for Active Duty Service members,
                        Veterans, Reservists, National Guard members, surviving
                        spouses, and certain other individuals who meet
                        eligibility requirements can use VA loans to purchase
                        homes or refinance existing mortgages. VA loans are
                        guaranteed by the Department of Veterans Affairs.
                    
    </p>
    <p>
      
                        VA loans are one of the only mortgages to offer 100%
                        financing (no down payment) for a home purchase. In
                        addition, VA loans offer refinance and cash out options
                        that can be used to pay off debt, fund major expenses,
                        make home improvements, or refinance a non-VA home loan
                        into a VA home loan with improved terms.
                    
    </p>
    <p>
      
                        There is also a streamlined option known as an Interest
                        Rate Reduction Refinance Loan (IRRRL) that provides
                        veterans with existing VA loans an opportunity to quickly
                        and easily refinance an existing VA loan. Underwriting
                        terms are very relaxed on an IRRRL an they allow veterans
                        or active duty to quickly and easily pursue a refinance
                        when market conditions improve and interest rates drop.
                    
    </p>
  </div>
</section>
`;
